### DISCUSSION TITLE
Sorry but I can't find the Standallone installation files

### DISCUSSION THREAD

**Discussion by LordKa-Berlin** [zanllp/infinite-image-browsing#735] [Q&A] [answered]
Hey sorry about the question, but I can't find the installation files for the Standallone installation.
“Download and install the program from the releases section on the right-hand side of the repository page. “ I have read, but I can't find it. Could someone explain this to a dummy in more detail?

**Comment by zanllp** [comment, marked as answer]
<img width="463" alt="image" src="https://github.com/user-attachments/assets/5a2a8fee-beae-4cf0-9e5e-afa90c7c278f">
<img width="1083" alt="image" src="https://github.com/user-attachments/assets/45da3282-4b84-4888-a3fb-f55c276f3476">
